samples/ballerina-by-example/examples.txt

```diff
@@ -56,6 +56,7 @@ Header Based Routing
 Produces/Consumes
 HTTP Sessions
 HTTP Client Connector
+HTTP Redirection
```

Comment:
This has to be "HTTP Redirects" .. the system needs to match the directory name of the sample .. each word wise ..